setup.py

```diff
@@ -48,7 +48,7 @@
     },
     python_requires='>=3.5,<3.9',
     install_requires=[
-        'pandas>=0.23.2,<1.1.0',
+        'pandas>=0.23.2',
```

Comment:
Could you also update `requirements-dev.txt`?

Reply:
Oh, thanks. My mistake